projects/rero/ng-core/src/lib/record/files/files.component.html

```diff
@@ -22,94 +22,48 @@ <h5 class="m-0" translate>Files</h5>
-              <a [href]="getFileUrl(file)" target="_blank" class="btn btn-sm btn-outline-primary ml-1" download>
-                <i class="fa fa-download mr-1"></i>{{ 'Download' | translate }}
-              </a>
-              <button class="btn btn-sm btn-outline-danger ml-1" (click)="delete(file)">
-                <i class="fa fa-trash mr-1"></i>{{ 'Delete' | translate }}
-              </button>
-            </div>
-          </div>
-          <div class="row mt-2" *ngIf="file.showInfo && !metadataForm.model">
-            <div class="col-lg-6" *ngIf="file.is_head">
-              <dl class="row mt-2 mb-0">
-                <ng-container *ngFor="let item of file.metadata | keyvalue">
-                  <ng-container *ngIf="!infoExcludedFields.includes(item.key) && item.value">
-                    <dt class="col-sm-4">{{ item.key | translate }}</dt>
-                    <dd class="col-sm-8">{{ item.value | translate }}</dd>
-                  </ng-container>
-                </ng-container>
-              </dl>
-            </div>
-            <div class="col-lg-6" [ngClass]="{ 'pl-5 text-muted': !file.is_head }">
-              <dl class="row mt-2 mb-0">
-                <dt class="col-sm-4" translate>Size</dt>
-                <dd class="col-sm-8">{{ file.size | filesize }}</dd>
-
-                <dt class="col-sm-4" translate>Mime type</dt>
-                <dd class="col-sm-8">{{ file.mimetype }}</dd>
-
-                <dt class="col-sm-4" translate>Checksum</dt>
-                <dd class="col-sm-8">{{ file.checksum }}</dd>
-
-                <dt class="col-sm-4" translate>Modified at</dt>
-                <dd class="col-sm-8">{{ file.updated | dateTranslate:'medium' }}</dd>
-              </dl>
-            </div>
-          </div>
+          <ng-core-record-file
+            [type]=type
+            [record]=record
+            [file]=file
+            [hasChildren]=hasChildren(file)
+            [infoExcludedFields]=infoExcludedFields
+            (deleteFile)=deleteFile(file)
+            (editMetadataFile)=editMetadataFile(file)
+            (manageFile)=manageFile(file)
+          ></ng-core-record-file>
         </li>
       </ng-container>
     </ul>
   </ng-container>
-  <ng-template #noFiles>
-  </ng-template>
+
   <div class="card-body" *ngIf="hasError || files.length === 0">
-    <div class="alert alert-info m-0" *ngIf="!hasError && files.length === 0" translate>No file found for this record.
+    <div class="alert alert-info m-0" *ngIf="!hasError && files.length === 0" translate>
+      No file found for this record.
     </div>
     <div class="alert alert-danger m-0" *ngIf="hasError" translate>An error occurred, files cannot be loaded.</div>
   </div>
-  <div class="card-footer text-center">
-    <button class="btn btn-sm btn-primary" (click)="manageFile(null)">
+  <div *ngIf=canAdd.can class="card-footer text-center">
+    <button class="btn btn-sm btn-primary" *ngIf=canAdd (click)="manageFile(null)">
```

Comment:
Is it necessary to have `*ngIf=canAdd` because there's already a div definition on the previous line?